gamestonk_terminal/cryptocurrency/due_diligence/glassnode_view.py

```diff
@@ -0,0 +1,200 @@
+from gamestonk_terminal.cryptocurrency.due_diligence.glassnode_model import (
+    get_active_addresses,
+    get_exchange_balances,
+    get_exchange_net_position_change,
+)
+
+
+def display_active_addresses(
+    asset: str, since: int, until: int, interval: str, export: str = ""
+) -> None:
+    """Display active addresses of a certain asset over time
+    [Source: https://glassnode.org]
+
+    Parameters
+    ----------
+    asset : str
+        Asset to search active addresses (e.g., BTC)
+    since : int
+        Initial date timestamp (e.g., 1_614_556_800)
+    until : int
+        End date timestamp (e.g., 1_614_556_800)
+    interval : str
+        Interval frequency (e.g., 24h)
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+
+    df_addresses = get_active_addresses(asset, interval, since, until)
+
+    if df_addresses.empty:
+        print("Error in glassnode request")
+    else:
+        plot_data(df_addresses, asset)
+    print("")
+
+    export_data(
+        export,
+        os.path.dirname(os.path.abspath(__file__)),
+        "active",
+        df_addresses,
+    )
+
+
+def display_exchange_net_position_change(
+    asset: str, exchange: str, since: int, until: int, interval: str, export: str = ""
+) -> None:
+    """Display 30d change of the supply held in exchange wallets.
+    [Source: https://glassnode.org]
+
+    Parameters
+    ----------
+    asset : str
+        Asset to search active addresses (e.g., BTC)
+    exchange : str
+        Exchange to check net position change (e.g., binance)
+    since : int
+        Initial date timestamp (e.g., 1_614_556_800)
+    until : int
+        End date timestamp (e.g., 1_614_556_800)
+    interval : str
+        Interval frequency (e.g., 24h)
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+
+    df_addresses = get_exchange_net_position_change(
+        asset, exchange, interval, since, until
+    )
+
+    if df_addresses.empty:
+        print("Error in glassnode request")
+    else:
+        _, ax1 = plt.subplots(figsize=(25, 7))
+
+        ax1.fill_between(
+            df_addresses[df_addresses["v"] < 0].index,
+            df_addresses[df_addresses["v"] < 0]["v"].values / 1e3,
+            np.zeros(len(df_addresses[df_addresses["v"] < 0])),
+            facecolor="red",
+        )
+        ax1.fill_between(
+            df_addresses[df_addresses["v"] >= 0].index,
+            df_addresses[df_addresses["v"] >= 0]["v"].values / 1e3,
+            np.zeros(len(df_addresses[df_addresses["v"] >= 0])),
+            facecolor="green",
+        )
+
+        ax1.set_ylabel(
+            f"30d change of {asset} supply held in exchange wallets [thousands]"
+        )
+        plt.title(
+            f"{asset}: Exchange Net Position Change - {'all exchanges' if exchange == 'aggregated' else exchange}"
+        )
+        plt.xlim([df_addresses.index[0], df_addresses.index[-1]])
```

Comment:
same here, can replace with ax.set_title() and ax.set_xlim().  (And going against what I just said, leaving the dates as plt. functions is fine for me now)